microcosm_flask/paging.py

```diff
@@ -84,10 +85,15 @@ def to_tuples(self):
             ("offset", self.offset),
             ("limit", self.limit),
         ] + [
-            (key, self.rest[key])
+            (key, self._qs_value_for_tuple(self.rest[key]))
             for key in sorted(self.rest.keys())
         ]
 
+    def _qs_value_for_tuple(self, value):
```

Comment:
Is there a reasonable spec we could add? Perhaps we could update https://github.com/globality-corp/microcosm-flask/blob/d18311d4e88bdb1f0483d5bbc44c49c805af236d/microcosm_flask/tests/test_paging.py#L23?